src/main/resources/templates/pebt/householdList.html

```diff
@@ -16,7 +16,9 @@
           <div class="boxed-content text--centered">
             <strong th:text="#{household-list.your-household}"></strong>
             <ul class="subflow-list list--bulleted">
-              <li class="spacing-below-15" th:text="${inputData.firstName} + ' ' + ${inputData.lastName} + ' (' + #{household-list.thats-you} + ')'"></li>
+              <th:block th:if="${T(org.homeschoolpebt.app.utils.SubmissionUtilities).applicantIsInHousehold(submission)} && !${T(org.homeschoolpebt.app.utils.SubmissionUtilities).isApplyingForSelf(submission)}">
```

Comment:
I think the `&&` and `!` have to be within the `${...}`

```suggestion
              <th:block th:if="${T(org.homeschoolpebt.app.utils.SubmissionUtilities).applicantIsInHousehold(submission) && !T(org.homeschoolpebt.app.utils.SubmissionUtilities).isApplyingForSelf(submission)}">
```